api.go

```diff
@@ -492,9 +492,26 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		}
 		r.processConfigurationLogEntry(&entry)
 	}
+
 	r.logger.Printf("[INFO] raft: Initial configuration (index=%d): %+v",
 		r.configurations.latestIndex, r.configurations.latest.Servers)
 
+	mismatchedIP := false
```

Comment:
Talked in person and we should be able to delete this. `consul operator raft list-peers` is the right thing to look at, and this will be incorrect until the reconcile loop fixes it up, but there's no need to fix the local node up like this. The local node will determine if it has a vote by ID - https://github.com/hashicorp/raft/blob/library-v2-stage-one/configuration.go#L145-L154.